appimage/build-appimage.sh

```diff
@@ -1,26 +1,20 @@
 #!/usr/bin/env bash
 
-# MIT License
```

Comment:
Probably we haven't rights to change the license here

Reply:
fixed, as well as other files where MIT license was overwritten